src/containers/audio-rewards-page/components/modals/ChallengeRewards.tsx

```diff
@@ -262,25 +309,48 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
           </div>
         </div>
       )}
-      {buttonLink && (
-        <Button
-          className={wm(styles.button)}
-          type={ButtonType.PRIMARY_ALT}
-          text={buttonInfo?.label}
-          onClick={goToRoute}
-          leftIcon={buttonInfo?.leftIcon}
-          rightIcon={buttonInfo?.rightIcon}
-        />
+      <div className={wm(styles.claimRewardWrapper)}>
+        {buttonLink && (
+          <Button
+            className={wm(styles.button)}
+            type={
+              isComplete && !isDisbursed
+                ? ButtonType.COMMON
+                : ButtonType.PRIMARY_ALT
+            }
+            text={buttonInfo?.label}
+            onClick={goToRoute}
+            leftIcon={buttonInfo?.leftIcon}
+            rightIcon={buttonInfo?.rightIcon}
+          />
+        )}
+        {challenge && isComplete && !isDisbursed && !hideClaimButton && (
+          <ClaimRewardButton
+            className={cn(wm(styles.button), {
+              [styles.disabled]: claimStatus !== ClaimStatus.NONE
+            })}
+            challengeId={modalType}
+            specifier={specifier}
+            amount={amount}
+            isDisabled={claimStatus !== ClaimStatus.NONE}
+            icon={icon}
+          />
+        )}
+      </div>
+      {displayClaimError && (
+        <div className={styles.claimError}>Oops, something’s gone wrong 😞</div>
```

Comment:
can we vendor this guy into the .emoji class structure we have so it's system independent https://github.com/AudiusProject/audius-client/tree/master/src/assets/fonts/emojis


![disappointed-face](https://user-images.githubusercontent.com/2731362/134469504-213c2c06-e50c-4335-8c8f-3e35397e8b69.png)